src/views/SystemIntake/Documents/UploadForm.tsx

```diff
@@ -29,13 +29,18 @@ import {
 import { CreateSystemIntakeDocumentInput } from 'types/graphql-global-types';
 import { fileToBase64File } from 'utils/downloadFile';
 import { documentSchema } from 'validations/systemIntakeSchema';
+import Pager from 'views/TechnicalAssistance/RequestForm/Pager';
 
 type DocumentUploadFields = Omit<CreateSystemIntakeDocumentInput, 'requestID'>;
 
+type UploadFormProps = {
+  type: 'admin' | 'requester';
```

Comment:
Do you see this around enough to pull it up like [RequestType](https://github.com/CMS-Enterprise/easi-app/blob/main/src/types/requestType.ts)?

Reply:
I couldn't find any other components with this prop, but I'll keep that in mind if we run into it again.